src/v2/Components/NavBar/Menus/UserMenu.tsx

```diff
@@ -22,10 +22,13 @@ export const UserMenu: React.FC = () => {
   const { trackEvent } = useTracking()
   const { mediator, user } = useContext(SystemContext)
 
-  const trackClick = event => {
-    const link = event.target
+  const trackClick = (event: React.MouseEvent<HTMLElement, MouseEvent>) => {
+    const link = event.currentTarget
+
+    if (!(link instanceof HTMLAnchorElement)) return
+
     const text = link.innerText
-    const href = link.parentNode.parentNode.getAttribute("href")
+    const href = link.href
```

Comment:
This was a weird little bug I noticed here: https://artsy.slack.com/archives/CP9P4KR35/p1594305193454600

If you happened to click on the icon, href would be undefined because of how this was bubbling. It's much clearer/ less error prone to stick the click handlers on the anchors directly.